static/js/jira-select-server.js

```diff
@@ -0,0 +1,18 @@
+$('.install-link').click(function (event) {
+  event.preventDefault();
+  const installationId = $(event.target).data('installation-id');
```

Comment:
I would just delete all of this for now. Not much sense having code in here that's going to change. Leave the event.preventDefault() and just delete the rest. When we start wiring up routing we can update this to query GitHubServerApps for the provided installationId and then either navigate to the screen with listed GH apps or to the auto/manual screen.

Reply:
The logic has been removed and a TODO has been added